src/extension.spec.ts

```diff
@@ -135,19 +133,32 @@ describe('extension', () => {
             });
         });
 
-        describe('setupRDBViewProviders', () => {
-            it('calls setupODC to create the odc instance if enabled', () => {
-                config.injectRdbOnDeviceComponent = true;
-                const spy = sinon.stub(extensionInstance, 'setupODC').returns({});
-                extensionInstance.setupRDBViewProviders(config);
-                expect(spy.calledOnce).to.be.true;
-            });
-
-            it('does not call setupODC if not enabled', () => {
-                config.injectRdbOnDeviceComponent = false;
-                const spy = sinon.stub(extensionInstance, 'setupODC').returns({});
-                extensionInstance.setupRDBViewProviders(config);
-                expect(spy.calledOnce).to.be.false;
+        describe('debugSessionCustomEventHandler', () => {
+            describe('BSChannelPublishedEvent', () => {
+                const e = {
+                    event: 'BSChannelPublishedEvent',
+                    body: {
+                        launchConfiguration: config
+                    }
+                };
+
+                afterEach(() => {
```

Comment:
Is this necessary? We're already doing the exact same thing on line 44. 

https://github.com/rokucommunity/vscode-brightscript-language/pull/401/files#diff-0d79d039c81b1141d7761f91b299b6eb9e7d71f8341fb305652f3bd475f77221R43

Reply:
Removed I was trying to debug why the test was failing and accidentally left in